assets/formol-baseline-logo.svg

```diff
@@ -0,0 +1,48 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="299" height="103" viewBox="27 27 244 27">

```

Comment:
@annabellou @gentooboontoo I've changed the `viewBox` of this svg to crop the bottom margin. 
Feel free if we should not or you have another idea (dedicated no-margin logo) ?

Reply:
@glepretre , I can give you logo without margins if you want, maybe it's simplier ?